feat(template): add new jinja filters (#3498)
Mergifyio/mergify-engine: docs/source/configuration.rst

when used in your configuration file — considering the pull request author
login is ``jd``.

Jinja2 filters are supported, you can build string from list for example with:

.. code-block:: jinja

    Approved by: @{{ approved-reviews-by | join(', @') }}

We also provide custom Jinja2 filters:

* ``markdownify``: to convert html to markdown:

.. code-block:: jinja

    {{ body | markdownify }}

* ``get_section(<section>)``: to extract one markdown section

.. code-block:: jinja

    {{ body | get_section("## Description") }}

.. note::

   You need to replace the ``-`` character by ``_`` from the :ref:`pull request
   attribute <attributes>` names when using templates. The ``-`` is not a valid
   character for variable names in Jinja2 template.

.. note::

>    By default, the html comment is stripped from ``body``, to get the
>    full body, you should use ``body-raw``.

Reviewer: ```suggestion
   By default, the HTML comments are stripped from ``body``. To get the
   full body, you can use the ``body_raw`` attribute.
```
